docs/_plugins/collapsible.rb

```diff
@@ -0,0 +1,47 @@
+module Jekyll
+  module Tags
+    class CollapseTag < Liquid::Block
+      def initialize(tag_name, block_options, liquid_options)
+        super
+        @title = block_options.strip
+      end
+
+      def render(context)
+        accordionID = context["accordionID"]
+        idx = context["collapsed_idx"]
+        collapsedID = "#{accordionID}-collapse-#{idx}"
+        headingID = "#{accordionID}-heading-#{idx}"
+
+        # increment for the next collapsible
+        context["collapsed_idx"] = idx + 1
+
+        site = context.registers[:site]
+        converter = site.find_converter_instance(::Jekyll::Converters::Markdown)
+        content = converter.convert(super)
+
+        output = <<~EOS
+          <div class="card" style="overflow:visible">
+            <div class="card-header" id="#{headingID}" style="padding:0px">
+              <h4 class="mb-0">
+                <button class="btn btn-link collapsed" data-toggle="collapse" data-target="##{collapsedID}" aria-expanded="false" aria-controls="#{collapsedID}">
```

Comment:
is `##` expected or a typo?

Reply:
it is expected. i am not sure how this all works, but `#{varName}` generates to `varValue` in HTML, while `##{varName}` generates to `#varValue`. the use of `#` in HTML appears for the `data-target` and `data-parent` values; when i remove it from the HTML, the accordion blocks don't expand/collapse

Reply:
sounds good. Thanks for the explanation